### ISSUE TITLE
Type error at offer/create

### ISSUE THREAD

**Issue by juliaanholcer** [CodeForPoznan/volontulo#719]
__Description:__

Type error occurs while adding new offer

__Current behaviour:__

While adding an offer, when start point is set to "already going" and end point is set to some date, an error occurs:
```
TypeError at /o/offers/create
unorderable types: NoneType() < datetime.datetime()
```
However an offer is added to database.

__Expected behaviour:__

Describe expected behaviour of application.

__Steps to reproduce:__

![screenshot1](https://user-images.githubusercontent.com/28116755/32011320-ba8f76ec-b9b4-11e7-9f69-c7798904074f.png)
![screenshot2](https://user-images.githubusercontent.com/28116755/32011342-c777b234-b9b4-11e7-8d88-5e3644c16cba.png)

**Comment by magul** [comment]
We had this bug before in backlog, but I will close the former one, as this one better described.

**Comment by jacekkalbarczyk** [comment]
Biorę.

**Comment by magul** [comment]
closed in #738
